os/arch/arm/src/amebasmart/amebasmart_usb.c

```diff
@@ -0,0 +1,803 @@
+/****************************************************************************
```

Comment:
Many of the variable/function requires cleanup and more comment explanation. But due to tight schedule, let's do it in a new PR.